sw/host/cryptotest/testvectors/parsers/wycheproof_ecdsa_parser.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python3
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import json
+import jsonschema
+import logging
+import sys
+
+from Crypto.Util.asn1 import DerSequence
+
+# Mapping from the curve names used by Wycheproof to those used by cryptotest
+EC_NAME_MAPPING = {
+    "secp256k1": "p-256",
+    "secp384r1": "p-384",
```

Comment:
nit: unless this is matching up with some other naming somewhere, I'd do
```suggestion
    "secp256k1": "p256",
    "secp384r1": "p384",
```
just to stay consistent with the naming in other parts of the crypto code.

Reply:
It's currently aligned with the ECDSA testvector schema that I defined in a previous PR, but that was an arbitrary decision based on how NIST was naming things. I'll add a commit that changes the naming scheme (since it will also add a small change to the NIST parser).

Currently the hash names also contain hyphens. Should I change those as well? `sha-256` to `sha256`?

Reply:
I'd say `sha256` is more consistent, but it's not a big deal; hash naming is less consistent across the cryptolib in general because e.g. SHA256 can also be called SHA2-256 in place where we want to distinguish with SHA3.